mytoyota/controller.py

```diff
@@ -1,23 +1,20 @@
 """Toyota Connected Services Controller """
-from __future__ import annotations
-
-from datetime import datetime
+from datetime import datetime, timedelta
 from http import HTTPStatus
 import logging
+import pprint
 from typing import Any
+from urllib import parse  # For parse query string, can this be done with httpx?
 
 import httpx
+import jwt
```

Comment:
The `jwt` module still needs to be added to pyproject.toml.
Otherwise you will receive the following error message, unless it is installed in the current env by chance.
`ModuleNotFoundError: No module named 'jwt'`

Reply:
Resolved with commit https://github.com/DurgNomis-drol/mytoyota/pull/242/commits/b2b60708b0a98087201aee4700a156e0e7edec3a